Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/runtime/conversion_generator.go

```diff
@@ -185,11 +185,15 @@ func isComplexType(reflection reflect.Type) bool {
 func (g *conversionGenerator) generateConversionsForMap(inType, outType reflect.Type) error {
 	inKey := inType.Key()
 	outKey := outType.Key()
+	g.addImportByPath(inKey.PkgPath())
```

Comment:
This already covers it? https://github.com/openshift/origin/blob/master/Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/runtime/deep_copy_generator.go#L154